src/ClientData/CallstackData.cpp

```diff
@@ -238,34 +244,56 @@ void CallstackData::FilterCallstackEventsBasedOnMajorityStart() {
       }
     }
 
-    static constexpr double kFilterSupermajorityThreshold = 0.6;
+    static constexpr double kFilterSupermajorityThreshold = 0.75;
```

Comment:
Is that modification intended? There should probably be then a comment. Is this value arbitrary?

Reply:
It is arbitrary, and it is intended, I wanted to make it stricter. Added a comment.